.travis.yml

```diff
@@ -2,5 +2,6 @@ language: node_js
 node_js:
   - "0.10"
   - "0.12"
-  - "4.0"
-  - "5.0"
\ No newline at end of file
+  - "4"
+  - "5"
+  - "stable"
```

Comment:
Any issues here?

Looks like 'stable' runs 5.2.0 and '5' runs 5.3.0, which is a bit odd. NVM stable for me locally is 5.3.0. '4' runs 4.2.4 which is appropriate.

Stable might be duplicate but it would run 6 when it comes out.  We could delete 5 from here.